COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,201 @@
+# io.js Collaborator Guide
+
+This document contains information for Collaborators of the io.js
+project regarding maintaining the code, documentation and issues.
+
+Collaborators should be familiar with the guidelines for new
+contributors in [CONTRIBUTING.md](./CONTRIBUTING.md) and also
+understand the project governance model as outlined in
+[GOVERNANCE.md](./GOVERNANCE.md).
+
+## Issues and Pull Requests
+
+Courtesy should always be shown to individuals submitting issues and
+pull requests to the io.js project.
+
+Collaborators should feel free to take full responsibility for
+managing issues and pull requests they feel qualified to handle, as
+long as this is done while being mindful of these guidelines, the
+opinions of other Collaborators and guidance of the TC.
+
+Collaborators may **close** any issue or pull request they believe is
+not relevant for the future of the io.js project. Where this is
+unclear, the issue should be left open for several days to allow for
+additional discussion. Where this does not yield input from io.js
+Collaborators or additional evidence that the issue has relevance, the
+issue may be closed. Remember that issues can always be re-opened if
+necessary.
+
+## Accepting Modifications
+
+All modifications to the the io.js code and documentation should be
+performed via GitHub pull requests, including modifications by
+Collaborators and TC members.
+
+All pull requests must be reviewed and accepted by a Collaborator with
+sufficient expertise who is able to take full responsibility for the
+change. In the case of pull requests proposed by an existing
+Collaborator, an additional Collaborator is required for sign-off.
+
+In some cases, it may be necessary to summon a qualified Collaborator
+to a pull request for review by @-mention.
+
+If you are unsure about the modification and are not prepared to take
+full responsibility for the change, defer to another Collaborator.
+
+Before landing pull requests, sufficient time should be left for input
+from other Collaborators. Leave at least 48 hours during the week and
+72 hours over weekends to account for international time differences
+and work schedules. Trivial changes (e.g. those which fix minor bugs
+or improve performance without affecting API or causing other
+wide-reaching impact) may be landed after a shorter delay.
+
+Where there is no disagreement (or discussion) amongst Collaborators,
```

Comment:
I think the "(or discussion)" is confusing here. What we're trying to say is "if there is no disagreement go ahead and land it" but this makes it sounds like what we're really looking for is a lack of input and discussion at all which I don't think we're advocating.